formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/services/FormstackService.scala

```diff
@@ -74,30 +75,41 @@ object FormstackService extends FormstackRequestService with LazyLogging {
     else
       decode[FormSubmissions](response.body)
   }
-
+  
+  val skippableErrorMessages = List(
+    "A valid submission id was not supplied",
+    "A valid form could not be found"
+  )
+
+  def isSkippableError(response: HttpResponse[String] ) : Boolean = !response.is2xx && skippableErrorMessages.exists(response.body.contains)
+
+  def decodeIfNotSkippableError[T: Decoder](response: HttpResponse[String]): Either[Throwable, Option[T]] = decode[T](response.body) match {
+    case Left(_) if isSkippableError(response) =>
+      logger.info(s"skipping response with status: ${response.statusLine}")
+      Right(None)
+    case Left(e) => Left(e)
+    case Right(submission) => Right(Some(submission))
+  }
+  
   private def getSubmissions(
     submissionIdEmails: List[SubmissionIdEmail],
     accountToken: FormstackAccountToken,
     encryptionPassword: String): Either[Throwable, List[Submission]] = {
-    submissionIdEmails.traverse { submissionIdEmail =>
+    val submissionResults: Either[Throwable, List[Option[Submission]]] = submissionIdEmails.traverse { submissionIdEmail =>
       val response =
         Http(s"https://www.formstack.com/api/v2/submission/${submissionIdEmail.submissionId}.json")
           .header("Authorization", accountToken.secret)
           .param("encryption_password", encryptionPassword)
           .asString
-
+          
       if(!response.is2xx) {
         logger.error(response.body)
       }
-
-      /* There are a couple of forms that the Formstack API can't seem to decrypt. There seems to be no way around this
-      *  so we capture this specific error and skip these forms. */
-      if(response.body.contains("An error occurred while decrypting the submission"))
-        Left(FormstackDecryptionError(s"${response.body} | submission id: ${submissionIdEmail.submissionId}"))
```

Comment:
just to sum up our previous conversation for future reference: 

Going from the comment in the code  it seems like the intention was to skip this error and keep going, but the current code wasn't doing doing that, it was just returning a custom error type that nothing picked up later.
We don't think this error should be skipped anyway, but if we later decide it should be we can add it to the list of skippable errors